src/reader.cr

```diff
@@ -18,6 +18,15 @@ end
 
 sql = File.read(ARGV[0])
 meta = get_metadata(sql)
-puts "def #{meta}"
-puts "\"#{parse(sql).strip}\""
-puts "end"
+
+heredoc = <<-SQL
+#{parse(sql).gsub("\"", "'").strip}
```

Comment:
Replacing all quotes to single-quote may not be a good idea. IMHO we shouldn't touch file contents. It should be remain as-is.

e.g.
```sql
SELECT * FROM users WHERE name = 'Fatih "fka" Akin'
```

Your code will break it.